RevenueCatUI/Views/LoadingPaywallView.swift

```diff
@@ -71,6 +71,9 @@ private extension LoadingPaywallView {
         },
         restorePurchases: {
             fatalError("Should not be able to purchase")
+        },
+        trackEvent: { _ in
+            fatalError("Should not track events from a loading paywall")
```

Comment:
This _might_ be crashing right now, but it's a good way to ensure that we don't send wrong events from this!

Reply:
I guess it will have to be replaced with a no-op.